src/WebJobs.Script.WebHost/App_Start/AutofacBootstrap.cs

```diff
@@ -17,9 +18,15 @@ internal static void Initialize(ContainerBuilder builder, WebHostSettings settin
             {
                 RootScriptPath = settings.ScriptPath,
                 RootLogPath = settings.LogPath,
-                FileLoggingEnabled = true
+                FileLoggingEnabled = true,
+                TraceWriter = settings.TraceWriter,
             };
 
+            if (settings.IsSelfHost && settings.TraceWriter != null)
```

Comment:
Can you add a comment here? Is your intent to disable all logging? Perhaps you don't want FileLoggingEnabled above as well?